module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php

```diff
@@ -276,10 +276,12 @@ public function getAvailability($overDriveId)
                 }
             } else {
                 $result->status = true;
-                $res->copiesAvailable ??= 0;
-                $res->copiesOwned ??= 0;
-                $res->numberOfHolds ??= 0;
-                $res->code = 'od_none';
+                if ($res) {
+                    $res->copiesAvailable ??= 0;
+                    $res->copiesOwned ??= 0;
+                    $res->numberOfHolds ??= 0;
+                    $res->code = 'od_none';
+                }
```

Comment:
Zero is better than an ugly debug error trace, but it's probably wrong information.  Ideal would be a clean error message?